docs/source/index.md

```diff
@@ -47,37 +47,30 @@ We have a number of client-side SDKs available for different languages.
 
 A number of "adapters" are available for some popular Inference and Vector Store providers. For other APIs (particularly Safety and Agents), we provide *reference implementations* you can use to get started. We expect this list to grow over time. We are slowly onboarding more providers to the ecosystem as we get more confidence in the APIs.
 
-**Inference API**
```

Comment:
I think this replacement isn't quite accurate because the former table lists individual API providers, while the replacement is a table of "distributions" (namely bundles of providers)

Reply:
Ah right, so should this table move to docs/source/distributions/selection.md instead? Thanks!